### PR TITLE
feature/SEAR-920-increase-grpc-max-receive-msg-size

### PR DESCRIPTION
**Pull request by nbkincaid**
**Issue Link**
https://spothero.atlassian.net/browse/SEAR-920

**Description**
Valid search search transient availability responses were breaking the grpc message size default limit. This bumps it up to 20mb from roughly 4.2mb.

This also adds a format make target and commits the initial formatted codebase.

### CODE REVIEW

**Review by greg-mclain — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -15,3 +15,4 @@ example_server
 coverage.txt
 jwt-cli
 example-server
+.idea/*
```
Comment (dj80hd) [inline .gitignore:18]: is it common to check this in vs put it in one's global .gitignore?

File: Makefile [hunk 2]
```diff
@@ -27,3 +27,6 @@ help: ## Prints this help command
 		sort | \
 		awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
+.PHONY: format
+format: ## formats the codebase
```
Comment (dj80hd) [inline Makefile:31]: nice.  should lint depend on this?

File: .gitignore (hunk 1 above)
Comment (nbkincaid) [reply .gitignore:18]: I'm not sure if we have a standard, but it seems to me its somewhat arbitrary in this case. Is there a downside you see?

File: Makefile (hunk 2 above)
Comment (nbkincaid) [reply Makefile:31]: I'm not sure, it depends on how we want to use it I'd say.

Comment (nbkincaid) [reply Makefile:31]: I'm actually going to break this out into its own pr, I'm thinking the codecov run on master is behind and it's affecting the mergability of this pr.

File: http/circuit_breaker_test.go [hunk 3]
```diff
@@ -48,7 +48,7 @@ func TestNewDefaultCircuitBreakerRoundTripper(t *testing.T) {
 			"host configuration is honored",
 			&mock.RoundTripper{ResponseStatusCodes: []int{http.StatusOK}, CreateErr: false},
 			map[string]circuit.Config{
-				"host": circuit.Config{},
+				"host": {},
```
Comment (dj80hd) [inline http/circuit_breaker_test.go:51]: oh cool, assignments to slice and map values have an implicit type!

**Pull request merged**